README.md

```diff
@@ -133,18 +140,60 @@ In addition, you might need to install Corepack. For more information checkout [
+> Note: This command will also open up Storybook locally. This provides you with a full list of available components that you can experiment with. Each component in Storybook also contains detailed documentation how to use them and the parameters that they can accept.
+
 ### Setup
-Note: This is not filled out yet!
+
+> Note: To be able to edit tokens, you must own a pro license of token studio
+
+1. First, open your Copy of the B01LER and bookmark Tokens Studio in the Plugins panel
+2. Next, launch Tokens Studio using 'New Empty File' since no tokens have been defined yet
+3. Connect your repository to Tokens Studio. First, generate and copy a new Personal Access Token in Github. You can use either Tokens (Classic) or 'Fine-grained tokens'. Next, use the copied token to add GitHub as a Sync Provider. Click the 'Settings tab' in Tokens Studio and in the 'Add new menu', select GitHub. Use the Personal Token that you copied to authenticate yourself and to connect your cloned B01LER repository.
+4. If the entered credential are correct you will be asked to Pull from git. Please accept this dialog. The tokens tab should now contain all tokens sets of B01LER available.
+
+Please refer to the 'Getting Started' section in our [Figma file](https://www.figma.com/file/C4vgEKz8mKyulJ4gm3Qdql/%F0%9F%AB%A7-%5BBLR%5D-The-B01LER?type=design&node-id=8311-4751&mode=design&t=SPtTkLRxRECGO4Ga-0) (the link to the Figma file above is exclusively for members of the Accenture organization, while those outside the organization can download the file [here](https://boilerds.com/boilerds/B01LER-v1-alpha%20release.fig)) for more detailed instructions on how to set up the library, configure Figma Tokens Studio and customize the project assets to meet your requirements.
+
 ...
 
-### Usage
-Note: This is not filled out yet!
+The 'Getting Started' section also explains the logic behind our token names and enable you to find and change the right tokens when rebranding B01LER or when you want to extend or change some functionality of a component.
 ...
 
 
 ## :white_check_mark: How to test
 Learn about our testing approach [here](./doc/TESTING.md).
 
+## :toolbox: How to build
+> Note: Each command must be run from the root directory of the project
+
+The B01LER UI Library can be built using the following command:
+
+   ```sh
+   $ yarn build:ui-library
+   ```
+
+B01LER can also be built with Storybook bundled alongside it which provides you with a sandbox to experiment with our components:
+
+   ```sh
+   $ yarn build:storybook
+   ```
+
+B01LER comes with an example vanilla JS application to demonstrate how easy it is to use components in your app. To build this example app, run the following command:
+
+   ```sh
+   $ yarn build:js-example
+   ```
+
+In the generated `dist` folder, the generated `index.html` file can be opened in the browser to preview our components.
+...
+
+## rocket: How to deploy
+Learn about our approach to deployment [here](./doc/DEPLOYMENT.md).
+...
```

Comment:
Remove this?

Reply:
Won't we need this for later?

Reply:
Sorry I only mean line 191, should've specified